js/.eslintrc.js

```diff
@@ -3,3 +3,6 @@
 // http://www.apache.org/licenses/LICENSE-2.0
 
 module.exports = require('@attic/eslintrc');
+
+// TODO: Include this in @attic/eslintrc.
+module.exports.rules['eqeqeq'] = ['error', 'allow-null'];
```

Comment:
After reading the rest of the PR it is also clear to me that using `== null` is the cowards way out. We should know what we want.